### DISCUSSION TITLE
Relationship with the  Credit Commons Society

### DISCUSSION THREAD

**Discussion by dilgreen** [CredComSoc/ProtocolWorkingGroup#11] [General]
This Protocol Working Group (PWG) has a background context, in the work of collaboration around all aspects of the Credit Commons Protocol (CCP/CredCom), so a little history is in order here.

Collaboration grew beyond Matthew getting occasional support from Dil (2019-20) into an active social group in March 2020. This group was looking more at ways of bringing CCP/CredCom into use contexts than at the Protocol per se, but the eventual direction was always ‘in the mix’ - which was for that collaboration to ‘birth’ two distinct projects. This ‘birthing’ took place in 2021: one project was explicitly about business-viable route-to-market work, which became [Mutual Credit Services (MCS)](http://mutualcredit.services),  the other, more or less explicitly a proto-Foundation, became the [Credit Commons Society (the CCS)](http://creditcommonssociety.org).

### Mutual Credit Services

While MCS sees its role as both a contributing supporter and an implementer of the CCP/CredCom, it does not consider that its ‘commercial’ status makes it appropriate for it to be any sort of ‘governor’ of the Protocol. If the CCP/CredCom is to be adopted as widely as possible in all sorts of ‘accounting for trade’ contexts, it must not be seen as being strongly directed by any particular commercial interests.

MCS looks to the CCS to provide a development context for the CCP/CredCom which strongly emphasises general ecosystem benefit as a deep ethic. MCS sees itself as a contributing supporter to the CCS.

### Credit Commons Society

The CCS sees itself as working towards becoming capable of fulfilling a rounded ‘Foundation’ role - looking after the development of the whole ecosystem around the CCP/CredCom - doing what it can to provide the conditions in which CCP/CredCom-based projects of all kinds can move forward together.

A key aspect of the CCS’ work at present is in developing its own Governance structure and processes. This is seen as important work, since, if the overall project gains traction, CCP/CredCom will begin to be a foundational tool for many Real World Economy projects - which will result in a wide range of pressures and demands. The urgency and force of these will be a sign of success - to be welcomed - but will also pose many currently unforeseen challenges. If these are to be effectively handled, it is important that a considered and culturally embedded Governance understanding be in place.

### CCS Governance
CCS is still developing its Governance model, but there are some clear foundations in place.

CCS is working to bring into coherence a collection of relevant frameworks.

- the cybernetics inspired Viable System Model (more particularly the the “Moral Modalities Framework’ developed by Trevor Hilder as a more easily applied outline of the VSM).
   - this is seen as providing a base understanding of what a viable system must both have in place, and be able to map onto...
- Elinor Östrom’s ‘Principles of Commons Governance’
    - these are seen as providing more specific conditions for effective governance, both of the organisation type CCS considers itself to be building, and for the kind of organisations the CCP is design to support.
- Chris Cook’s ‘Nondominium’ approach to implementing these principles in formal Agreements
- Sociocracy as a set of practices and processes to enable decentralised and effective decision-making. the Sociocratic approach is seen as:
    - allowing effective subsidiarity and autonomy to working groups, without loss of coordination or clarity.
    - acting to counter ‘ossification’ of organisational power structures (and thus capacity to imagine).

Emerging from this work is a fairly simple structure:
- a General Circle, which acts to:
    - Develop and periodically review the Direction, Vision and Ethics statements of the CCS
    - coordinate several working Circles
        - A Communications Circle
        - An Action Research Circle
        - A Protocol Circle
    - notice Issues requiring attention, 
        - surfacing those so that they can be understood by the membership
        - acting so that proposals for how such Issues will be attended to are developed, refined, and considered for adoption
        - ensuring adopted proposals can be acted upon.

### What this means for the Protocol Working Group
Effectively, the CCS asks the Protocol Working Group to consider itself as one of the three permanent ‘working’ Circles of the CCS - what the CCS calls the ‘Protocol Circle’. *Of course, contributors to the PWG are not compelled to agree to this. See the next post in this discussion for more on this.*

In a [draft proposal](https://docs.google.com/document/d/1DtsEif2DIL6V1Fs3shlnMF-BiB1Q6ahxpRCcmyZgTq4/edit?usp=sharing) (still subject to refinement before it will be considered for adoption by the CCS - in which context it is important to state that the PWG will be a key participant in the process), the Protocol Circle’s ‘Domain’ is described thus;

#### To consider how best to bring the Credit Commons Protocol  (itself) to readiness for mass adoption, through such means as it should see fit, which it is assumed will include versions of:
- Active open source developer group
- Robust, well documented reference implementation
- Validation of the Protocol for a variety of use cases
- Improved understanding of federation mechanisms
.. and so on

The CCS has also drafted ‘expectations’ for working Circles’, as follows;

#### Each Working Circle should:
- Develop and periodical review its own Direction and Vision statements, together with any additions to the Ethics of the CCS it considers important
- Work in the spirit of the adopted Ethics
- Document its proceedings, decisions and operational work as agreed with the Anchor/General Circle
- Work with the Anchor/General Circle through the sociocratic ‘bi-directional representation’ method. This is expected to deliver:
   - Clear communication into the Anchor/General Circle as to issues the Working Circle thinks need to be addressed by the Anchor/General Circle, with particular reference to;
      - Needs of the Working Circle which it cannot provide for itself
      - Requests for clarification as to issues around Direction 
      - Clear communication from the Anchor/General Circle

The ‘bi-directional representation’ method referred to here is simply this - that each Working Circle chooses two ‘representatives’ - one whose role is to represent the working Circle in the General Circle, and one whose role is to represent the General Circle within the Working circle.

**Comment by dilgreen** [comment]
The current members of this Working Group are invited to review this - currently loosely defined, but still significant - proposition as to how it considers itself to relate the the CCS, and discuss here.

If we are following Sociocratic processes, things could go something like this:

1. Anyone who needs it can ask for clarification 
2. Once clarification is sufficient, then there may be a need to create a specific proposal (which would include a period to first review)
3. Once a proposal is ready, then consent is sought (consent need not indicate approval, just acceptance of the intent of others, no hard block)
4. Consent creates a policy.

**Comment by tfwoodroofMCS** [comment]
A couple of comments:

> The ‘bi-directional representation’ method referred to here is simply this - that each Working Circle chooses two ‘representatives’ - one whose role is to represent the working Circle in the General Circle, and one whose role is to represent the General Circle within the Working circle.

Each Working Circle chooses at least one 'representative' to the General Circle (referred to as a 'delegate'), with the other half of the bi-directional linking being the 'co-ordinator', who can be chosen from either the Working Circle or the General Circle (the latter being preferred in more hierarchical implementations).

> **To consider how best to bring the Credit Commons Protocol (itself) to readiness for mass adoption, through such means as it should see fit, which it is assumed will include versions of:**
...
...
Validation of the Protocol for a variety of use cases
...

The Action Research Circle (another permanent Working Circle) should be able to help supply such use-cases, suggesting the need for cross-representation to/from this circle to there. This is another recognised role in sociocracy.
